website/components/search-bar/style.css

```diff
@@ -0,0 +1,8 @@
+.g-search {
+  width: calc(100% - 2rem);
+  max-width: 600px;
+
+  & .c-hits .ais-Highlight-highlighted {
+    /* Add custom background color */
```

Comment:
Is this still a TODO?

Reply:
I can prob remove, it was a placeholder for design critiques, but I'll ❌